tests/heif/heif_im2im.c

```diff
@@ -15,6 +17,9 @@ int main()
 	int size = 0;
 	CuTestImageResult result = {0, 0};
 
+	if (!gdTestAssertMsg(heif_have_decoder_for_format(heif_compression_HEVC) && heif_have_encoder_for_format(heif_compression_HEVC), "HEVC codec support missing from libheif\n"))
+		return 0;
```

Comment:
you want to return `77` instead.  this will mark the test as SKIP.

Reply:
Ain't skipping with `ctest`, is there any other return code?

Reply:
hrm, autotest respects `77` already.  looks like we need to opt-in to it in cmake:
https://cmake.org/cmake/help/latest/prop_test/SKIP_RETURN_CODE.html

i just pushed 16fab2f35a26 which should make this work for us too.  i did a brief test and it seemed to work.

Reply:
Thanks for pushing that.